kratos/modified_shape_functions/modified_shape_functions.cpp

```diff
@@ -292,7 +294,7 @@ namespace Kratos
                 }
 
                 aux_grad_sh_func_cond = prod(aux_grad_sh_func_exp, rPmatrix);
-                rInterfaceShapeFunctionsGradientsValues.push_back(trans(aux_grad_sh_func_cond));
+                rInterfaceShapeFunctionsGradientsValues[i_interface*n_int_pts + i_gauss] = trans(aux_grad_sh_func_cond);
```

Comment:
@rubenzorrilla please check that I put the right index here.

Reply:
It's correct! :1st_place_medal:

Reply:
Yes, it's the right one.